src/sql/parts/query/editor/messagePanel.ts

```diff
@@ -116,6 +119,66 @@ export class MessagePanel extends ViewletPanel {
 				this.state.collapsed = !this.isExpanded();
 			}
 		});
+		this.controller.onKeyDown = (tree, event) => {
+			if (event.ctrlKey) {
```

Comment:
will this work on mac where we expect it to be cmd+a?

Also does it play nicely if the user adds a custom keyboard shortcut for ctrl+a or ctrl+c? I would expect ctrl+a or ctrl+c to work here if the console has focus and otherwise do the mapped action

Reply:
Taken care of in the new implementation.